### ISSUE TITLE
issue: conflict in version of pytest dependency

### ISSUE THREAD

**Issue by tserg** [vyperlang/vyper#2561]
### Version Information

* vyper Version (output of `vyper --version`): 0.3.1
* OS: linux
* Python Version (output of `python --version`): 3.9.0

### What's your issue about?

Running `python setup.py test` results in the following error:

```
pkg_resources.ContextualVersionConflict: (pluggy 1.0.0 (/home/tserg/vyper/.eggs/pluggy-1.0.0-py3.9.egg), Requirement.parse('pluggy<1.0,>=0.12'), {'pytest'})
```

### How can it be fixed?

Update the `pytest` version in `setup.py` from `"pytest>=5.4,<6.0"` to `"pytest>=6.2.5,<7.0"`.

This dependency issue is resolved in v6.2.5. Refer to this [https://github.com/pytest-dev/pytest/releases/tag/6.2.5](link) for more details.

**Comment by charles-cooper** [comment]
Was this fixed in https://github.com/vyperlang/vyper/pull/2563?

**Comment by tserg** [comment]
> @tserg Was this fixed in #2563?

Yes, correct.

**Comment by charles-cooper** [comment]
Great, thank you
